src/set_config.py

```diff
@@ -38,6 +38,13 @@ def file_objects():
     return os.getenv("OBJECT_DIR")
 
 
+def always_display_namespace():
```

Comment:
This function returns either a boolean (False) or a string - os.getenv().
It should probably always return a boolean - either True or False